testdata/fix/nginx-v1alpha2/fn-config.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-config
+  annotations:
+    config.kubernetes.io/local-config: "true"
```

Comment:
This annotation won't be needed (m3 issue tracking this).

Reply:
Is this only specified for function configs ? or users do it for non fn-config resources as well. I will follow up on this and change it after we push that change.